run.py

```diff
@@ -1,7 +1,7 @@
 from tutor import app, db
 
 
-from tutor.routing import *
+from tutor.routes import home
```

Comment:
the fact that home is not use in this file will cause issues in the future when you try to introduce Pylint, but we can let it be for now

Reply:
Yes I know it will cause problems with the linters - but there is another solution? 
I tried to find another way, but all reference I saw, including the other repos on that username, imports the routing file.